client/components/wp-login-footer/README.md

```diff
@@ -0,0 +1,9 @@
+WP Login Footer
```

Comment:
Is `WP` necessary?

Reply:
That makes sense. I removed WP from everywhere.